src/test/java/co/rsk/bitcoinj/core/StoredBlockTest.java

```diff
@@ -168,41 +168,45 @@ public void serializeCompactV2_forMoreThan32bytesChainWork_throwsException() {
     }
 
     @Test
-    public void serializeCompactV2_forZeroChainWork_works() {
-        testSerializeCompactV2(ZERO_CHAIN_WORK);
+    public void serializeAndDeserializeCompactV2_forZeroChainWork_works() {
+        testSerializeAndDeserializeCompactV2(ZERO_CHAIN_WORK);
     }
 
     @Test
-    public void serializeCompactV2_forSmallChainWork_works() {
-        testSerializeCompactV2(SMALL_CHAIN_WORK);
+    public void serializeAndDeserializeCompactV2forSmallChainWork_works() {
+        testSerializeAndDeserializeCompactV2(SMALL_CHAIN_WORK);
     }
 
     @Test
-    public void serializeCompactV2_for8BytesChainWork_works() {
-        testSerializeCompactV2(EIGHT_BYTES_WORK_V1);
+    public void serializeAndDeserializeCompactV2for8BytesChainWork_works() {
+        testSerializeAndDeserializeCompactV2(EIGHT_BYTES_WORK_V1);
     }
 
     @Test
-    public void serializeCompactV2_for12ByteChainWork_works() {
-        testSerializeCompactV2(MAX_WORK_V1);
+    public void serializeAndDeserializeCompactV2for12ByteChainWork_works() {
+        testSerializeAndDeserializeCompactV2(MAX_WORK_V1);
     }
 
     @Test
-    public void serializeCompactV2_forTooLargeWorkV1_works() {
-        testSerializeCompactV2(TOO_LARGE_WORK_V1);
+    public void serializeAndDeserializeCompactV2forTooLargeWorkV1_works() {
+        testSerializeAndDeserializeCompactV2(TOO_LARGE_WORK_V1);
     }
 
     @Test
-    public void serializeCompactV2_for32BytesChainWork_works() {
-        testSerializeCompactV2(MAX_WORK_V2);
+    public void serializeAndDeserializeCompactV2for32BytesChainWork_works() {
+        testSerializeAndDeserializeCompactV2(MAX_WORK_V2);
     }
 
-    private void testSerializeCompactV2(BigInteger chainWork) {
```

Comment:
I would probably make these parametrized tests, either way LGTM

Reply:
As said in the other PR, this project is using junit 4. We need to agree on updating it to junit5. I agree on doing it but I think this out of scope of this project now.